grunt-tasks/cssmin.js

```diff
@@ -10,9 +10,9 @@ module.exports = function(grunt) {
     combine: {
       files: {
         // output
-        "<%= cssPath %>/dist/webcompat.min.css": [
+        "<%= cssPath %>/webcompat.min.css": [
           // input
-          "<%= cssPath %>/webcompat.dev.css"
+          "<%= cssPath %>/webcompat.min.css"
```

Comment:
just to be sure I understand it right... we parse the dev.css to min.css in the sass task and then minify in the cssmin task later on, right?

Reply:
Yes, I tried to use the `output: "compressed"` option in grunt-sass, but it didn't work out. So I stayed with cssmin, even if it's ugly for the Sass to CSS process 😒

Reply:
we also have the option to minify via https://www.npmjs.com/package/grunt-purifycss as this is already build in.

You can find the discussion here: https://github.com/webcompat/webcompat.com/issues/2331

Reply:
So, this is all set up already and a part of the grunt workflow. Just the minify param is missing right now. We'd then need to remove the cssmin module. Did I forget anything?

Reply:
Correct. If purifycss takes care of minification as well, then we should only drop the cssmin package and adjust the input filepath for purifycss.